CITATIONS.md

```diff
@@ -12,30 +12,42 @@
 
 - [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
 
-  > Andrews, S. (2010). FastQC: A Quality Control Tool for High Throughput Sequence Data [Online].
+> Andrews, S. (2010). FastQC: A Quality Control Tool for High Throughput Sequence Data [Online].
+
+- [gzrt](https://www.urbanophile.com/arenn/hacking/gzrt/gzrt.html)
+
+> Renn, A. M. (2013). The gzip Recovery Toolkit [Online].
+
+- [Wipertools](https://github.com/mazzalab/fastqwiper)
+
+> Mazza, T. (2025). Wipertools. [![DOI](https://zenodo.org/badge/325529896.svg)](https://doi.org/10.5281/zenodo.14176147){: style="float:right"}
```

Comment:
```suggestion
> MazzaLab, Tommaso Mazza, & Jose F Oviedo. (2025). mazzalab/fastqwiper: v1.1.5 (v1.1.5). Zenodo. [https://doi.org/10.5281/zenodo.14774039](https://doi.org/10.5281/zenodo.14774039)
```

On the left of the Zenodo record, you can see the recommended citation format: 

![image](https://github.com/user-attachments/assets/6fa74c34-aba2-4242-b287-6b5afbc2658e)